app/isomorphic/components/layouts/footer/styles.m.css

```diff
@@ -0,0 +1,13 @@
+.footer {
+  clear: both;
+  text-align: center;
+  font-size: 20px;
+  background: black;
+  color: white;
```

Comment:
why not variables ?

Reply:
These styles are very skeleton-ish and will be replaced by actual styles. And we are going to add the CSS architecture changes which we're following in Vikatan after once we're confident on it.